### ISSUE TITLE
Can't publish the dotnet core 2.2 webapi project

### ISSUE THREAD

**Issue by onlyanup** [KishoreIthadi/DotNet-Core-Essentials#44]
When I execute the command **DCE: Publish** and select **_<my-project-name.csproj>_** file, it throws this error :
DCE: The selected project is not a core or standard framework
![image](https://user-images.githubusercontent.com/37064591/61639345-f1129e80-acb8-11e9-80b2-d141894e4711.png)

This is the content of my csproj file:
```
<Project Sdk="Microsoft.NET.Sdk.Web">
  <PropertyGroup>
    <LangVersion>8.0</LangVersion>
  </PropertyGroup>
  <PropertyGroup>
    <TargetFramework>netcoreapp2.2</TargetFramework>
    <RuntimeFrameworkName>Microsoft.AspNetCore.App</RuntimeFrameworkName>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.App"/>
    <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1"/>
    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.6"/>
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.6"/>
    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="2.2.6"/>
    <PackageReference Include="Newtonsoft.Json" Version="12.0.2"/>
  </ItemGroup>
</Project>

```

**Comment by rougiain** [comment]
Can confirm the same on 2.2 Console App

**Comment by ManojVeeranala** [comment]
@onlyanup @rougiain 
Apologies for the late reply
I am unable to reproduce the issue, I have tested this with DCE v0.0.7 with your csproj file.

**Comment by onlyanup** [comment]
@ManojVeeranala The latest release is 0.0.6 and I'm sure that's where I had the issue. Now if you have fixed it in 0.0.7 then thats good. I will check it in the coming days and update accordingly.

![image](https://user-images.githubusercontent.com/37064591/64941102-5f23ae00-d883-11e9-9014-0510a209b081.png)

**Comment by ManojVeeranala** [comment]
@onlyanup  We released a new version today. Please update your extension and check. If the issue persists please open the issue again.
Thanks @onlyanup @rougiain
